Gemfile.lock

```diff
@@ -185,4 +185,4 @@ DEPENDENCIES
   watir
 
 BUNDLED WITH
-   2.3.6
+   2.5.23
```

Comment:
Will this cause issues on hosts where this version isn't possible?  I don't recall bundlers behavior when the lock file wants a newer version than what is being used.

Reply:
This is the highest version available for Ruby 3. That's the lowest version of Ruby we support, so I think so.